doc/examples/introduction.ipynb

```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Not sure if we wanted to leave these notebooks un-executed in the repo? (@benbovy I vaguely remember that this was the case?)

Reply:
You should choose on this I suppose! Alternatively I've been switching my docs over from already-run notebooks to using `myst-nb` to be compiled by readthedocs, which I like a lot better. I can help switch this over to that if you want to give it a try.

Reply:
+1 for using `myst-nb`, if you want to give it a try that would be awesome!

Reply:
@benbovy I started to switch over to `myst-nb` to execute the docs notebooks, but I see that you are already doing that using `nbsphinx`! I actually didn't realize you could do that with `nbsphinx`. I can still change it over to `myst-nb` if you want, but I am not sure if there is any benefit to it over `nbsphinx`.